starlite/testing.py

```diff
@@ -368,8 +384,14 @@ def _create_scope(
             user=user,
             auth=auth,
             query_string=urlencode(query_params, doseq=True).encode() if query_params else b"",
-            path_params=[],
+            path_params=path_params,
             client=(self.server, self.port),
+            state=state,
+            asgi=ASGIVersion(spec_version="3.0", version="3.0"),
```

Comment:
I'm not sure this is the right way to build `ASGIVersion`. Please let me know if there's a better way